src/WebJobs.Script/Utility.cs

```diff
@@ -879,6 +881,32 @@ public static bool IsResourceAzureBlobWithoutSas(Uri resourceUri)
             return true;
         }
 
+        public static string GetSasTokenExpirationDate(Uri resourceUri)
+        {
+            // Screen out URLs with an SAS token
+            var queryParams = HttpUtility.ParseQueryString(resourceUri.Query);
+            if (!string.IsNullOrEmpty(queryParams[SasTokenExpirationDate]))
+            {
+                return queryParams[SasTokenExpirationDate];
+            }
+
+            return null;
+        }
+
+        public static string GetSasTokenExpirationDateFromSasSignature(string azureWebJobsStorage)
```

Comment:
Locally (not in core tools) I run with `UseDevelopmentStorage=true` -- will this throw?